components/BaseContentController.php

```diff
@@ -336,4 +340,63 @@ protected function prepareRequestParams(array $requestParams, string $formName,
 
         return $requestParams;
     }
+
+    protected function updateContent($activeRecord, $data): bool
+    {
+        if (!($activeRecord instanceof ContentActiveRecord)) {
+            return false;
+        }
+
+        if (empty($data['content']) || !is_array($data['content'])) {
+            return true;
+        }
+
```

Comment:
Could we handle other Content generic stuff here?

e.g.

- archived (yes / no)
- Pin to Top
- Visibility

Reply:
@luke- Ok, I have implemented the updating fields: `visibility`, `archived`, `pinned`, `locked_comments` - https://github.com/humhub/rest/pull/86/commits/0efdaece6d998c62a9033dc18cf4cae204b2a043.